terraform/applications/careers/providers.tf

```diff
@@ -12,3 +12,20 @@ provider "aws" {
     }
   }
 }
+
+# some objects, like lambdas at edge, must be deployed in east
```

Comment:
Is this due to an AWS restriction of some sort?

Reply:
yeah, lambdas for cloudfront have to be deployed in east. 'global' objects like cloudfront usually have to have us-east-1 defined.  This used to be true for IAM as well, that your botoclient/aws cli had to be set to VA.